### ISSUE TITLE
Line not working

### ISSUE THREAD

**Issue by AlgusDark** [emmanuelrosa/erosanix#15]
Hi, first of all, thanks for this very cool repo. I'm still learning nix and I'm trying to install line on my nixos.

I'm having the following error:
```
$ nix run github:emmanuelrosa/erosanix#line
error: hash mismatch in fixed-output derivation '/nix/store/xw79w356gwfchyrxwxdf71djlcw64v09-LineInst.exe.drv':
         specified: sha256-OZVWkgRA3idGY2lvgOhBqm2eHzJNZFBDxuZNZJZTMDc=
            got:    sha256-3dluIkOincyogSu89+L5O3oC6U6h/FUJPale+JhJtBw=
error: 1 dependencies of derivation '/nix/store/nfi8rlvpa2msiqr3406cy9r10p1swgrk-line-8.7.0.3302.drv' failed to build
```

I have no clue on how I could fix this, I did try to override the sha256 of the line exe file with but with no succes.

Any help would be really appreciated.
Thanks!

**Comment by emmanuelrosa** [comment]
Yes, Line is tricky to update. One sec...

**Comment by emmanuelrosa** [comment]
Fixed in 6dba2ff84ce5982fad0921b4082908a8a9131a98

Note that LINE is finiky on Wine.

It's written in Qt, so in theory the LINE devs can make a Linux version, but... one can only dream.

If I recall, one of the quirks is that closing the window doesn't exit the app. You need to click on the button in the lower left corner and select "Exit."

Also, the app takes forever to load.

**Comment by AlgusDark** [comment]
> It's written in Qt, so in theory the LINE devs can make a Linux version, but... one can only dream.

Yeah, I know, I used to work there but they always had very weird decisions. Like when they closed the web version. Or their "open source contributions" that were more image than actual help :(

Thanks for the fix.
